client/SDL/sdl_disp.cpp

```diff
@@ -138,7 +138,7 @@ static BOOL sdl_disp_check_context(void* context, sdlContext** ppsdl, sdlDispCon
 	if (!context)
 		return FALSE;
 
-	sdl = (sdlContext*)context;
+	sdl = reinterpret_cast<sdlContext*>(context);
```

Comment:
Should be a `static_cast`

Reply:
no, as the types are pointers to different structs. (just try it ;))

Reply:
`context` is typeless (`void*`) so a `static_cast` is perfectly fine - just try it :-)

Reply:
ah ^^
yes, indeed this instance the parameter is `void*` (did have the issues with `reinterpret_cast` with the ones that have `rdpContext*` as argument